vars/kolaTestIso.groovy

```diff
@@ -1,102 +1,35 @@
 // Run kola testiso
 // Available parameters:
-//     arch:               string  -- the target architecture
 //     cosaDir:            string  -- cosa working directory
 //     extraArgs:          string  -- extra arguments to pass to `kola testiso`
-//     extraArgs4k:        string  -- extra arguments to pass to 4k `kola testiso`
-//     extraArgsMultipath: string  -- extra arguments to pass to multipath `kola testiso`
-//     extraArgsUEFI:      string  -- extra arguments to pass to UEFI `kola testiso`
-//     scenarios:          string  -- scenarios to pass to `kola testiso`
-//     scenarios4k:        string  -- scenarios to pass to `kola testiso`
-//     scenariosMultipath: string  -- scenarios to pass to `kola testiso`
-//     scenariosUEFI:      string  -- scenarios to pass to `kola testiso`
-//     skipMetal4k:        boolean -- skip metal4k image
-//     skipMultipath:      boolean -- skip multipath tests
-//     skipUEFI:           boolean -- skip UEFI tests
```

Comment:
Actually, I think there's one last PR we need for the pipeline. It currently uses `skipUEFI` here: https://github.com/coreos/fedora-coreos-pipeline/blob/fe2d907471cd7c73b01b9eca8c3ba4d4b1b1ab44/jobs/build.Jenkinsfile#L365. We'll need to convert that to something like `extraArgs: "--denylist-test *.uefi*"`?

Reply:
How about adding `skipUEFI` here too?

Reply:
SGTM! Let's redocument it as an option here?